src/main/java/com/minecolonies/coremod/util/AchievementUtils.java

```diff
@@ -38,14 +38,15 @@ public static void syncAchievements(@NotNull final Colony colony)
 
         for (@Nullable final EntityPlayer player : lPlayer)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "player" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1481)